data/json/requirements/melee.json

```diff
@@ -546,7 +549,11 @@
         [ "lucern_hammer", -1 ],
         [ "2h_club_kanabo_studded", -1 ],
         [ "2h_club_kanabo_spiked", -1 ],
-        [ "lucern_hammer", -1 ]
+        [ "lc_aztec_sword", -1 ],
```

Comment:
```suggestion
        [ "lucern_hammer", -1 ],
        [ "lc_aztec_sword", -1 ],
```
This will delete the lucerne hammer from the great hammers requirement, despite it being a great hammer.

Reply:
it's a duplicate

Reply:
Ah, true.